base::ClampToRange expects a float for the first argument but std::fabs returns
a double. Cast to a float.

--- ui/native_theme/native_theme_base.cc	2020-09-28 10:07:35.485919210 +1000
+++ ui/native_theme/native_theme_base.cc	2020-09-28 10:07:17.867786053 +1000
@@ -1483,7 +1483,7 @@
   SkScalar min_diff =
       base::ClampToRange((hsv1[1] + hsv2[1]) * 1.2f, 0.28f, 0.5f);
   SkScalar diff =
-      base::ClampToRange(fabs(hsv1[2] - hsv2[2]) / 2, min_diff, 0.5f);
+      base::ClampToRange((float)fabs(hsv1[2] - hsv2[2]) / 2, min_diff, 0.5f);
 
   if (hsv1[2] + hsv2[2] > 1.0)
     diff = -diff;
